include/networkit/auxiliary/Log.hpp

```diff
@@ -73,28 +75,12 @@ void setLogLevel(const std::string &logLevel);
  */
 std::string getLogLevel();
 
-namespace Settings {
-
-LogLevel getLogLevel();
-void setLogLevel(LogLevel p = LogLevel::info);
-
-void setPrintTime(bool b);
-bool getPrintTime();
-
-void setPrintLocation(bool b);
-bool getPrintLocation();
-
-void setLogfile(const std::string &filename);
-}
```

Comment:
We need to implement these functions by delegating to the `GlobalState` ones (to avoid API breaks).